modules/product/src/ProductAttributeValueStorage.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Drupal\commerce_product;
+
+use \Drupal\commerce\CommerceContentEntityStorage;
+
+/**
+ * Defines storage handler for product attribute value entities.
+ */
+class ProductAttributeValueStorage extends CommerceContentEntityStorage implements ProductAttributeValueStorageInterface {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function loadByPropertiesSortByWeight(array $values = []) {
```

Comment:
Maybe loadByAttributeSortByWeight? Allows us to skip using buildPropertyQuery() for cleaner code